clusterloader2/cmd/clusterloader.go

```diff
@@ -31,31 +30,31 @@ import (
 )
 
 var (
-	clusterConfig  config.ClusterConfig
-	testConfigPath string
+	clusterLoaderConfig config.ClusterLoaderConfig
 )
 
 func initClusterFlags() {
-	flag.StringVar(&clusterConfig.KubeConfigPath, "kubeconfig", "", "Path to the kubeconfig file")
-	flag.IntVar(&clusterConfig.NodeCount, "nodes", -1, "Number of nodes in the cluster")
+	flag.StringVar(&clusterLoaderConfig.ClusterConfig.KubeConfigPath, "kubeconfig", "", "Path to the kubeconfig file")
+	flag.IntVar(&clusterLoaderConfig.ClusterConfig.NodeCount, "nodes", -1, "Number of nodes in the cluster")
```

Comment:
I think you can get rid of this altogether (as we diuscused offline)

Reply:
Done.